reference-architecture/aws-ansible/ansible.cfg

```diff
@@ -0,0 +1,19 @@
+# config file for ansible -- http://ansible.com/
+# ==============================================
+[defaults]
+#callback_plugins = ../openshift-ansible/ansible-profile/callback_plugins
+forks = 50
+host_key_checking = False
+hostfile = inventory/aws/hosts/ec2.py
+remote_user = ec2-user
+gathering = smart
```

Comment:
This line should be removed as well.

You'll probably want to set the following:
```
retry_files_enabled = false
nocows = true
```

Otherwise, lgtm.
@abutcher any thoughts?